book/controller.rst

```diff
@@ -438,7 +438,7 @@ If you want to redirect the user to another page, use the ``redirectToRoute()``
         return $this->redirectToRoute('homepage');
 
         // redirectToRoute is equivalent to using redirect() and generateUrl() together:
-        // return $this->redirect($this->generateUrl('homepage'), 301);
```

Comment:
you are right @hhamon, but to have an example of a status code is not bad imo, what about changing `301` to `302` ??